lib/board.rb

```diff
@@ -0,0 +1,43 @@
+class Board
+  attr_accessor :board
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @board = Array.new ['', '', '', '', '', '', '', '', '']
+  end
+
+  def display_board
+    puts "#{@board[0]}(1) | #{@board[1]} (2)  |#{@board[2]}(3)"
+    puts '______________________'
+    puts "#{@board[3]}(4) | #{@board[4]} (5)  |#{@board[5]}(6)"
+    puts '______________________'
+    puts "#{@board[6]}(7) | #{@board[7]} (8)  |#{@board[8]}(9)"
```

Comment:
- [ ] [MANDATORY] No `puts/print` statements are allowed outside of the `main.rb` file.
Hint: You may assign the output to a variable here and then `puts` it in the `main.rb` file.